src/iccpd/src/iccp_cmd_show.c

```diff
@@ -99,6 +99,7 @@ int iccp_mclag_config_dump(char * *buf,  int *num, int mclag_id)
         state_info.role = csm->role_type;
 
         str_size = MCLAGDCTL_PORT_MEMBER_BUF_LEN;
+        len = 0;
```

Comment:
How does this fix the issue? `len` is initialized to 0 on line 48 and doesn't appear to be modified before this line.

Reply:
In line 61, we list for each `csm`, if we get 4 `enabled_po` in the first `csm`, and then the `len` is modified in line 115, the `len` has not been reinitialized to zero before we turn to the sceond `csm`, and then the second `csm's lif_po->name` could not be correctly copied to `state_info.enabled_po` in line 111, the field "MCLAG Interface" could not be displayed in the second `csm`. So, the len should be reinitialized to 0 in each `csm`.

Reply:
Thanks for the explanation. I missed that this is inside the `LIST_FOREACH` macro/loop.